### PR TITLE
Add navbar content section

### PR DESCRIPTION
**Pull request by wacoo**
- Add navbar
- Separate functionalities with menu
- Add a contact page
- Notification label

### CODE REVIEW

File: static/js/nav.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+});
+
+function showDefaultTab() {
+  add.style.display = 'none';
+  list.style.display = 'block';
+  contact.style.display = 'none';
+  btnList.className = 'active';
+}
+
+showDefaultTab();
```
Comment (danmainah) [inline static/js/nav.js]: - [ ] @Team great work adding the three tabs required for this app, however, they are all getting displayed at the same time as shown below kindly consider fixing the error causing all of them to be displayed at once
# Your page
image

# The Design required
image

Comment (danmainah) [reply static/js/nav.js]: - [ ] [optional] @Team Your design is missing the date and time section, kindly consider adding it to achieve the same design, as the design given 
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @wacoo , @paulshaun-kdtk and @Frelly0 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Added all Three content sections - Books list, Add book form and Contact info.
✔️ Professional written README file with all sections required.


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] @Team it would be better if you added the other authors' info in the author section in the readme file. This will make the readme have all the contributors' information.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job implementing the requested changes.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**